.github/actions/install_deps/action.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Install dependencies'
+description: "This action installs required dependencies."
+inputs:
+  type:
+    description: 'Type of installation'
+  additional_packages:
+    description: 'Additional packages to install'
+runs:
+  using: "composite"
+  steps:
+    - if: inputs.type != 'docker'
+      id: sudo
+      run: echo "::set-output name=sudo::sudo"
+      shell: bash
```

Comment:
We can avoid this if the release job uses `sudo`.

@mliszcz is there a problem with using `sudo` in docker images?

Reply:
OK great. Let's try installing `sudo` before calling the action and test if it works as expected. Thanks for the quick reply :)

Reply:
I tested a release and it seemed to work well. Would you mind testing it as well just as a sanity check? Only if you have the time of course, otherwise it's not worries: https://github.com/dundargoc/neovim/releases/tag/nightly